tests/Integration/ClientTest.php

```diff
@@ -4,58 +4,58 @@
 
 namespace Werkspot\KvkApi\Tests\Integration;
 
-use GuzzleHttp\Client;
 use PHPUnit\Framework\TestCase;
-use Werkspot\KvkApi\Api\Profile\Company;
-use Werkspot\KvkApi\Client\Adapter\Guzzle;
-use Werkspot\KvkApi\Client\Adapter\Guzzle\Exception\NotFoundException;
-use Werkspot\KvkApi\Client\Authentication;
-use Werkspot\KvkApi\Client\Endpoint;
-use Werkspot\KvkApi\Client\Search\ProfileQuery;
-use Werkspot\KvkApi\ClientFactory;
+use Werkspot\KvkApi\Client\Profile\Company;
+use Werkspot\KvkApi\Http\Search\ProfileQuery;
+use Werkspot\KvkApi\Http\Endpoint\Testing;
+use Werkspot\KvkApi\KvkClientFactory;
 
 /**
  * @large
  */
 final class ClientTest extends TestCase
 {
-    private const USERNAME = 'testourapis';
-    private const PASSWORD = 'testourapis';
-
     /**
      * @test
      * @dataProvider getKvkNumbers
      */
     public function getProfile(int $kvkNumber): void
```

Comment:
I think we should be more descriptive in this test method. what about `getProfile_successfullyGetCompaniesForKvkNumber`?

Reply:
I think for testing the happy flow `getProfile` is fine (since we should get the Profile)